pkg/build/strategies/layered/layered.go

```diff
@@ -150,18 +151,20 @@ func (builder *Layered) Build(config *api.Config) (*api.Result, error) {
 		buildResult.BuildInfo.FailureReason = utilstatus.NewFailureReason(utilstatus.ReasonGenericS2IBuildFailed, utilstatus.ReasonMessageGenericS2iBuildFailed)
 		return buildResult, errors.New("builder image name cannot be empty")
 	}
-
+	timeStart = time.Now()
 	if err := builder.CreateDockerfile(config); err != nil {
```

Comment:
> why? a failed step still has a start and finish time...

i would have to add the function again here and it would double the amount of code.
```
buildResult.BuildInfo.StepInfo = utilstatus.AddStepInfo(buildResult.BuildInfo.StepInfo, api.CreateDockerfileStep, timeStart)
```

a failed step does have a start/stop time, but is it relevant to the user to provide the build timing for a step that failed?